src/ConnectionAutomaton.cpp

```diff
@@ -345,11 +343,11 @@ void ConnectionAutomaton::processFrameImpl(
 void ConnectionAutomaton::onTerminal(
     folly::exception_wrapper ex,
     StreamCompletionSignal signal) {
-  auto thisPtr = this->shared_from_this();
-  auto movedEx = folly::makeMoveWrapper(ex);
-  runInExecutor([thisPtr, movedEx, signal]() mutable {
-    thisPtr->onTerminalImpl(movedEx.move(), signal);
-  });
+  runInExecutor([
+    thisPtr = this->shared_from_this(),
+    movedEx = std::move(ex),
```

Comment:
nit: do we need the moved prefix here?  What are our conventions for names with lambdas?